pkg/config/config.go

```diff
@@ -64,19 +64,32 @@ type Config struct {
 
 // TODO: https://github.com/jbrekelmans/kube-compose/issues/64
 // nolint
-func New() (*Config, error) {
-	fileName := "docker-compose.yml"
-	data, err := ioutil.ReadFile(fileName)
-	if err != nil {
+func New(file *string) (*Config, error) {
+	var data []byte
+	var err error
+	var fileName string
+	if file != nil {
+		if *file == "" {
```

Comment:
I think we don't need this empty string check anymore, because the ioutil.ReadFile will already produce error.

Reply:
I agree with the error being handled with the ioutil.ReadFile however I think the error message is more clear if it's empty as opposed to the wrong dir/file. If we really want I can generalize the error on the ioutil.ReadFile?